datasets/us/ss_wanted/us_ss_wanted.yml

```diff
@@ -0,0 +1,53 @@
+title: "United States Secret Service Most Wanted Fugitives"
+entry_point: crawler.py
+prefix: us-ss-wanted
+summary: "Wanted fugitivies listed on United States Secret Service website."
+description: >
+  This dataset publishes the details of the most wanted fugitives listed on the United States Secret Service website.
+
+  Individuals in this list are involved in one or more financial crime, including but not limited to counterfeiting, 
+  bank fraud, hacking, crypto and credit card fraud. 
+
+  The dataset includes details such as the fugitive's name, date of birth, aliases, summary of the criminal case, and relevant links to them.
+
+url: https://www.secretservice.gov/investigations/mostwanted
+load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
+coverage: monthly
```

Comment:
The `coverage` should have a `start` with the date this data was first crawled otherwise the crawler will not run.

Reply:
this is a good point to remind that CI has to be green, i.e. pass, before we can merge things. So please check the CI run and make sure it's green.